### DISCUSSION TITLE
Help accessing dashboard over VPN

### DISCUSSION THREAD

**Discussion by ddyer8** [tagyoureit/nodejs-poolController#521] [Q&A] [answered]
I'd like to be able to access my pool dashboard away from home.  I now have wireguard VPN setup and running on the rpi4 controlling my pool, but I'm not sure how to route to the other ports (5150 dashboard, 8080 REM, 8086 influxDB) so that I can access over VPN. I'm hoping it's something simple, but I don't know much about network configuration and access.

**Comment by rstrouse** [comment, marked as answer]
You only need 5150 then enable the proxy on the dashPanel connections tab

**Reply by tagyoureit** [reply]
If you really mean a VPN then you should have secure access back to your network on the VPN. In this case you shouldn't need to worry about any ports. 

If you mean exposing this to the internet then you should be very careful. We don't do any hardening of the app to prevent against intrusions.  It is not recommended.

**Comment by ddyer8** [comment]
It’s VPN. I was able to get it working using the IP address instead of the hostname.

Thank you,
Dustin
